src/enums/input-status.js

```diff
@@ -0,0 +1,3 @@
+export const EDITING = 'editing';
```

Comment:
I wouldn't put this into src/enums. Who is going to reuse this? It is internal state enum of the ec-inline-input-field, keep it in its folder.

Reply:
The file has been removed.